src/diffusers/schedulers/scheduling_utils.py

```diff
@@ -142,6 +142,8 @@ def from_pretrained(
             return_unused_kwargs=True,
             **kwargs,
         )
+        # _commit_hash
+        config.pop("_commit_hash", None)
```

Comment:
Yea, if commit hash becomes a return value instead of added on to config, we don't have to do this. This could be a bit confusing for someone reading for the first time